src/clarity-icons/shapes/basic-shapes.ts

```diff
@@ -3,7 +3,6 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-
 /* TODO: BasicShapes is deprecated and will be removed in 0.9.0 */
```

Comment:
Thanks @hippee-lee, it shouldn't affect the docs. And yes, BasicShapes and EssentialShapes have the same shapes. BasicShapes is deprecated and will be removed in 0.9.0.